src/portable/status.rs

```diff
@@ -388,13 +389,24 @@ async fn get_remote_async(instances: Vec<String>) -> anyhow::Result<Vec<RemoteSt
     Ok(result)
 }
 
+async fn get_cloud(cloud_client: &CloudClient) -> anyhow::Result<Vec<RemoteStatus>> {
+    match crate::cloud::ops::list(cloud_client).await {
+        Ok(rv) => Ok(rv),
+        Err(e) if e.is::<TimeoutError>() => {
```

Comment:
I think any error in cloud code could be processed the same way. I.e. connection refused, or 500 can be the same warning.

Reply:
Note #869